OpenAnatomyExport/OpenAnatomyExport.py

```diff
@@ -469,14 +469,20 @@ def addModelToRenderer(self, inputModelNode, outputModelNode, boostGouraudColor=
     outputPolyData.GetPointData().GetNormals().SetName("NORMAL")
     outputModelNode.SetAndObservePolyData(outputPolyData)
 
+    transformOutputModelToWorld = vtk.vtkGeneralTransform()
+    slicer.vtkMRMLTransformNode.GetTransformBetweenNodes(inputModelNode.GetParentTransformNode(), None, transformOutputModelToWorld)
+    transformToWorldFilter = vtk.vtkTransformPolyDataFilter()
```

Comment:
We should not add another vtkTransformPolyDataFilter. Instead, you can add the RAS-to-LPS transform to the other transforms by calling `transformOutputModelToWorld->Scale(-1.0, -1.0, 1.0)`. You just need to figure out if you need to set premultiply or postmultiply for the general transform (probably premultiply).

Reply:
This one:
```
    transformOutputModelToWorld = vtk.vtkGeneralTransform()
    transformOutputModelToWorld.PostMultiply()
    slicer.vtkMRMLTransformNode.GetTransformBetweenNodes(inputModelNode.GetParentTransformNode(), None, transformOutputModelToWorld)
    transformOutputModelToWorld.Scale(-1.0, -1.0, 1.0) #ras2lpsTransform
    transformToWorldFilter = vtk.vtkTransformPolyDataFilter()
    transformToWorldFilter.SetTransform(transformOutputModelToWorld)
    transformToWorldFilter.SetInputData(outputPolyData)
```

And this one:
```
    transformOutputModelToWorld = vtk.vtkGeneralTransform()
    transformOutputModelToWorld.PreMultiply()
    slicer.vtkMRMLTransformNode.GetTransformBetweenNodes(inputModelNode.GetParentTransformNode(), None, transformOutputModelToWorld)
    transformOutputModelToWorld.Scale(-1.0, -1.0, 1.0) #ras2lpsTransform
    transformToWorldFilter = vtk.vtkTransformPolyDataFilter()
    transformToWorldFilter.SetTransform(transformOutputModelToWorld)
    transformToWorldFilter.SetInputData(outputPolyData)
```

Both produced the correct exported result. 
I spent an hour wondering why? 
I knew the first one was correct, and I couldn't understand why changing the transforms order did not alter the result...
I checked the documentation, all appeared well...
Then I realized about this line -> https://github.com/Slicer/Slicer/blob/a5f75351073ef62fd6198d9480d86c0009d70f9b/Libs/MRML/Core/vtkMRMLTransformNode.cxx#L523
So, it may be important on the docstring of `vtkGeneralTransform::GetTransformBetweenNodes` to inform that multiply order of the result is always overwritten to `PostMultiply`

Reply:
It does not harm to add more information to the documentation especially if it might have helped you then it may help others in the future, too. Could you submit a pull request with the suggested documentation change?

Reply:
I guess this PR of SlicerOpenAnatomy can be merged

> It does not harm to add more information to the documentation especially if it might have helped you then it may help others in the future, too. Could you submit a pull request with the suggested documentation change?

Here is the [PR](https://github.com/Slicer/Slicer/pull/6840). I wanted to make a fast change using the github's web editor but it ended taking more time that doing it on a VS Code, since the web editor doesn't warn you about the actions that check commits mergeability I did a lot of PRs that did not succeeded the commit message check action